neural_compressor/adaptor/torch_utils/smooth_quant.py

```diff
@@ -898,16 +905,27 @@ def get_nodes(self, traced_model, op_types=['Linear']):
                     break
         return nodes
 
-    def get_prev_absorb_layer(self, nodes):
+    def get_prev_absorb_layer(self, nodes, dict_parent_kind=None):
         prev_absorb_layer = []
         for node in nodes:
             parent = get_parent(node)
+            parent_scopeName = parent.scopeName()
             while 1:
                 if parent.kind() in self.skip_ops_to_find_absorb:
                     parent = get_parent(parent)
                     continue
                 if parent.kind() in self.could_absorb_layers:
-                    prev_absorb_layer.append(parent)
+                    if dict_parent_kind:
+                        parent_out_kinds = set(dict_parent_kind[parent_scopeName])
+                        parent_out_kinds.discard('aten::size')
+                        if parent_out_kinds == parent_out_kinds.intersection(self.could_absorb_layers):
```

Comment:
self.could_absorb_layers is not correct, need to change to support_layers or the logic I told you